src/main/java/uk/gov/justice/digital/delius/service/OffenderService.java

```diff
@@ -169,6 +170,7 @@ public OffenderLatestRecall getOffenderLatestRecall(Long offenderId) {
         Offender offender = offenderRepository.findByOffenderId(offenderId)
                 .orElseThrow(() -> new NotFoundException("Offender not found"));
         uk.gov.justice.digital.delius.jpa.standard.entity.Event activeCustodialEvent = convictionService.getActiveCustodialEvent(offender.getOffenderId());
+        uk.gov.justice.digital.delius.jpa.standard.entity.Custody custody = custodyService.findCustodyFromCustodialEvent(activeCustodialEvent);
```

Comment:
Not sure we need a service to this, isn't just `Optional.of(activeCustodialEvent.getDisposal()).flatMap(d -> d.getCustody()).orElseThrow()`  

in other words the model contains it without doing further lookups?